projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -45,10 +43,9 @@ function register_block() {
 		return;
 	}
 
-	if (
-		( defined( 'IS_WPCOM' ) && IS_WPCOM )
-		|| ( ( new Connection_Manager( 'jetpack' ) )->has_connected_owner() && ! ( new Status() )->is_offline_mode() )
```

Comment:
Here we were also checking for offline mode. With this change, that check is going away. Maybe we would need to add it to `should_enable_monetize_blocks_in_editor`?

Reply:
@jeherve I am probably be wrong here, but I thought the `$manager->is_connected()` check in `should_enable_monetize_blocks_in_editor` would imply `!( new Status() )->is_offline_mode()` (if it is connected, then it is not offline)...

Reply:
Oh. `is_offline_mode` is actually the development mode...
```
	 * This was formerly called "Development Mode", but sites "in development" aren't always offline/localhost.
```
So I guess it is safe to include it in `should_enable_monetize_blocks_in_editor`

Reply:
Done in 4e335dca98f3a315e9a8479862935ac7dcf97f9b